src/main.py

```diff
@@ -64,17 +70,28 @@ def task1(conn):
 
       # let's be kind to the user and not make it case sensitive
 
-      if option.lower() == "n" and showing < batches:
-        # next page
-        showing += 1
-        clear()
-        continue
+      if option.lower() == "n":
+        if showing < batches:
+            # next page
+            showing += 1
+            clear()
+            continue
+        else:
+          # no next page
+          print("\nThere are no more pages. Enter P to go back a page, or choose a paper number.")
+          continue
```

Comment:
Why do you not clear in the event that the page goes out of bounds? Visually, not doing so can defeat the purpose of having the clear at all.